mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -43,11 +43,32 @@ fn from_thin_edge_json_with_timestamp(
     Ok(serializer.into_string()?)
 }
 
+/// Converts from thin-edge Json to c8y_json for child device
+pub fn from_thin_edge_json_with_child(
+    input: &str,
+    child_id: &str,
+) -> Result<String, CumulocityJsonError> {
+    let timestamp = WallClock.now();
+    let c8y_vec = from_thin_edge_json_with_child_with_timestamp(input, timestamp, child_id)?;
+    Ok(c8y_vec)
+}
+
+fn from_thin_edge_json_with_child_with_timestamp(
+    input: &str,
+    default_timestamp: DateTime<FixedOffset>,
```

Comment:
```suggestion
    timestamp: DateTime<FixedOffset>,
```

Wondering why it was called default in the `from_thin_edge_json_with_timestamp` method as well.